### PR TITLE
Updating, showing, and deleting users

### PR DESCRIPTION
**Pull request by ducquanghuynh0111**
ready
nh ch p man hinh 2018-12-10 luc 21 42 15

### CODE REVIEW

**Comment by ducquanghuynh0111** [comment]
ready

**Comment by nvtanh** [comment]
commented

2 commit nữa rồi e

**Comment by ducquanghuynh0111** [comment]
ready

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -1,11 +1,28 @@
...
     <nav>
       <ul class="nav navbar-nav navbar-right">
         <li><%= link_to t(".home"), root_path %></li>
         <li><%= link_to t(".help"), helf_path %></li>
-        <li><%= link_to t(".login"), signup_path %></li>
+        <% if logged_in? %>
+          <li><%= link_to t(".users"), users_path %></li>
+          <li class="dropdown">
+            <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+              Account <b class="caret"></b>
```
Comment (chien-pv) [inline app/views/layouts/_header.html.erb]: I18n nhé

File: app/views/sessions/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+      <%= f.label :email %>
+      <%= f.email_field :email, class: "form-control" %>
+
+      <%= f.label :password %>
+      <%= f.password_field :password, class: "form-control" %>
+
+      <%= f.label :remember_me, class: "checkbox inline" do %>
+        <%= f.check_box :remember_me %>
+        <span>Remember me on this computer</span>
```
Comment (chien-pv) [inline app/views/sessions/new.html.erb]: I18n

File: app/views/shared/_error_messages.html.erb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<% if @user.errors.any? %>
+  <div id="error_explanation">
+    <div class="alert alert-danger">
+      The form contains <%= pluralize(@user.errors.count, "error") %>.
```
Comment (chien-pv) [inline app/views/shared/_error_messages.html.erb]: I18n

**Comment by chien-pv** [comment]
commented

**Comment by ducquanghuynh0111** [comment]
ready

**Comment by ducquanghuynh0111** [comment]
ready

**Pull request merged**